src/Contact/LocalRelationship/Factory/LocalRelationship.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * @copyright Copyright (C) 2010-2021, the Friendica project
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace Friendica\Contact\LocalRelationship\Factory;
+
+use Friendica\Core\L10n;
+use Friendica\Core\Protocol;
+use Friendica\Model\Contact;
+use Psr\Log\LoggerInterface;
+
+class LocalRelationship extends \Friendica\BaseFactory implements \Friendica\Capabilities\ICanCreateFromTableRow
+{
+	/** @var L10n */
+	private $l10n;
```

Comment:
I don't see any usage of `$this->l10n`

Reply:
It's in an upcoming part of this refactoring, I can remove it for now.

Reply:
I think it would be better to have `L10n` calls just at the `Module` namespace, but I know that this would lead to variable based translation strings inside `t()` which are currently unsupported :-/

Reply:
Indeed, I'm working to move the `Contact::createFromProbe` method and it contains several `L10n->t()` calls.